plover/oslayer/keyboardcontrol.py

```diff
@@ -17,47 +17,30 @@
 """
 
 import sys
+import importlib
 
-KEYBOARDCONTROL_NOT_FOUND_FOR_OS = \
-        "No keyboard control module was found for os %s" % sys.platform
 
 if sys.platform.startswith('linux'):
-    from plover.oslayer import xkeyboardcontrol as keyboardcontrol
+    module_name = 'xkeyboardcontrol'
 elif sys.platform.startswith('win32'):
-    from plover.oslayer import winkeyboardcontrol as keyboardcontrol
+    module_name = 'winkeyboardcontrol'
 elif sys.platform.startswith('darwin'):
-    from plover.oslayer import osxkeyboardcontrol as keyboardcontrol
+    module_name = 'osxkeyboardcontrol'
 else:
-    raise Exception(KEYBOARDCONTROL_NOT_FOUND_FOR_OS)
+    raise Exception("No keyboard control module was found for os %s" % sys.platform)
 
+keyboardcontrol = importlib.import_module('.' + module_name, __package__)
```

Comment:
I still think that this is cleaner in the original form. Importlib is pretty obscure (and might make grepping in the source code harder)